main.js

```diff
@@ -1,223 +1,638 @@
-  sex: 'MEN\'s',
-  shoeImage: 'images/RUN/RUN NIKE AIR ZOOM TERRA KIGER 4 MEN\'S RUNNING SHOE $125 Style: 880563-001 Black Volt Hyper-Turquoise White.jpg'
-},
-{
-  brand: 'NIKE',
-  style: 'SB AIR MAX BRUIN VAPOR',
-  category: 'SKATE SHOES',
-  price: 100,
-  styleNumber: '882097-001',
-  color: 'Black/White/Cool-Grey',
-  numberColors: 3 + ' ' + 'Colors',
-  sex: 'MEN\'s',
-  shoeImage: 'images/SKATE/SKATE NIKE SB AIR MAX BRUIN VAPOR MEN\'S SKATEBOARDING SHOE $100 Style- 882097-001 Black White White Cool-Grey.jpg'
-},
-{
-  brand: 'NIKE',
-  style: 'SB BLAZER LOW GT',
-  category: 'SKATE SHOES',
-  price: 75,
-  styleNumber: '704939-018',
-  color: 'Gunsmoke/Gum Light/Orange Blaze/Black Spruce',
-  numberColors: 1 + ' ' + 'Colors',
-  sex: 'MEN\'s',
-  shoeImage: 'images/SKATE/SKATE NIKE SB BLAZER LOW GT MEN\'S SKATEBOARDING SHOE $75 Style- 704939-018 Gunsmoke Gum Light Orange-Blaze Black-Spruce.jpg'
-},
-{
-  brand: 'NIKE',
-  style: 'SB DUNK HIGH PRO',
-  category: 'SKATE SHOES',
-  price: 100,
-  styleNumber: '854851-010',
-  color: 'Dark Grey/Black/White/White',
-  numberColors: 3 + ' ' + 'Colors',
-  sex: 'MEN\'s',
-  shoeImage: 'images/SKATE/SKATE NIKE SB DUNK HIGH PRO MEN\'S SKATEBOARDING SHOE $100 Style- 854851-010 Dark-Grey Black White White.jpg'
-},
-{
-  brand: 'NIKE',
-  style: 'SB KOSTON 3 HYPERFEEL',
-  category: 'SKATE SHOES',
-  price: 150,
-  styleNumber: '819673-017',
-  color: 'Black/Yellow Strike/Gum/White',
-  numberColors: 2 + ' ' + 'Colors',
-  sex: 'MEN\'s',
-  shoeImage: 'images/SKATE/SKATE NIKE SB KOSTON 3 HYPERFEEL MEN\'S SKATEBOARDING SHOE $150 Style- 819673-017 Black Yellow-StrikeGum White.jpg'
-},
-{
-  brand: 'NIKE',
-  style: 'JORDAN SUPER FLY 5 PO',
-  category: 'BALL SHOES',
-  price: 140,
-  styleNumber: '881571-110',
-  color: 'White/White/Black',
-  numberColors: 2 + ' ' + 'Colors',
-  sex: 'MEN\'s',
-  shoeImage: 'images/BALL/BALL JORDAN SUPER FLY 5 PO MEN\'S BASKETBALL SHOE $140 Style: 881571-110 White White Black.jpg'
-},
-{
-  brand: 'NIKE',
-  style: 'KOBE A.D. NXT',
-  category: 'BALL SHOES',
-  price: 200,
-  styleNumber: '916832-710',
-  color: 'Volt/White/White',
-  numberColors: 1 + ' ' + 'Colors',
-  sex: 'MEN\'s',
-  shoeImage: 'images/BALL/BALL KOBE A.D. NXT MEN\'S BASKETBALL SHOE $200 Style: 916832-710 Volt White White.jpg'
-},
-{
-  brand: 'NIKE',
-  style: 'LEBRON XIV AGIMAT',
-  category: 'BALL SHOES',
-  price: 175,
-  styleNumber: '852402-400',
-  color: 'Coastal Blue/Star Blue/White',
-  numberColors: 2 + ' ' + 'Colors',
-  sex: 'MEN\'s',
-  shoeImage: 'images/BALL/BALL LEBRON XIV AGIMAT MEN\'S BASKETBALL SHOE $175 Style: 852402-400 Coastal-Blue Star-Blue White.jpg'
-},
-{
-  brand: 'NIKE',
-  style: 'KYRIE 2 iD',
-  category: 'BALL SHOES',
-  price: 200,
-  styleNumber: '852402-666',
-  color: 'Black/Pure Gold/White',
-  numberColors: 2 + ' ' + 'Colors',
-  sex: 'MEN\'s',
-  shoeImage: 'images/BALL/BALL NIKE KYRIE 2 iD_Black Gold White .jpg'
-}
+/* eslint-disable no-unused-vars */
```

Comment:
You don't want to use this directive in your projects. Unused vars are bad. What problem is this solving for you?